src/main.ts

```diff
@@ -222,10 +230,25 @@ export class Main {
     this.backgroundWindow = null;
     app.quit();
   }
+ private checkForDuplicate(args){
```

Comment:
would it make sense to create a `helpers/helpers.ts` and move this method there? or into existing `helpers/helpers.js` if possible?

Reply:
yeah make sense, move this function to `helpers/helpers.js`